doto-frontend/src/components/pages/Settings/__snapshots__/SettingsPage.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<SettingsPage /> component being rendered Make sure render matches snapshot 1`] = `ReactWrapper {}`;
```

Comment:
Same thing here as the calendar component, its supposed to render the settings component not just a react wrapper